package.json

```
@@ -55,7 +55,7 @@
     "node-fetch": "2"
   },
   "peerDependencies": {
-    "gaxios": "^5.0.2",
-    "googleapis": "^109.0.1"
+    "gaxios": "^6.0.3",
+    "googleapis": "^131.0.0"
```

How about instead we do this?

```suggestion
    "gaxios": ">= 5",
    "googleapis": ">= 109"
```

---

Perfect - that will be nice and future-proof.